components/dropdown/dropdown-content-mixin.js

```diff
@@ -735,6 +745,21 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		return null;
 	}
 
+	_handleFocusTrapEnter() {
+		if (this.__applyFocus && !this.noAutoFocus) {
+			const content = this.__getContentContainer();
+			const focusable = getFirstFocusableDescendant(content);
+			if (focusable) {
+				// bumping this to the next frame is required to prevent IE/Edge from crazily invoking click on the focused element
+				requestAnimationFrame(() => focusable.focus());
+			} else {
+				content.setAttribute('tabindex', '-1');
+				content.focus();
+			}
+		}
+		this.dispatchEvent(new CustomEvent('d2l-dropdown-focus-enter', { bubbles: true, composed: true }));
```

Comment:
Does this need to bubble and be composed? Seems like the consumer could just wire up directly to the dropdown-content if they care about it.